docs/application/dotnet/guides/nui/widget.md

```diff
@@ -0,0 +1,255 @@
+# NUI Widget Application
+
+[NUIWidgetApplication](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.NUIWidgetApplication.html) helps you to create Tizen.NET widget applications, which are small versions of Tizen.NET applications on the home screen.
+You can launch these applications on the home screen when you select them in the **Add Widget** menu.
```

Comment:
@tscholb I think this sentence is also requested to be re-checked.

https://github.com/Samsung/tizen-docs/pull/988#discussion_r365080085

> "You can launch these applications on the home screen when you select them in the Add Widget menu." is not suitable for this guide because your rephase assumes "Add Widget" menu on the home screen.

Would you please elaborate it?